insecure/integration/functional/test/gossipsub/rpc_inspector/validation_inspector_test.go

```diff
@@ -424,7 +460,7 @@ func TestValidationInspector_UnknownClusterId_Detection(t *testing.T) {
 			corruptlibp2p.CorruptGossipSubConfigFactoryWithInspector(corruptInspectorFunc)),
 	)
 	idProvider.On("ByPeerID", victimNode.ID()).Return(&victimIdentity, true).Maybe()
-	idProvider.On("ByPeerID", spammer.SpammerNode.ID()).Return(&spammer.SpammerId, true).Times(3)
+	idProvider.On("ByPeerID", spammer.SpammerNode.ID()).Return(&spammer.SpammerId, true).Times(4)
```

Comment:
can there be more docs here explaining why we're expecting 4 returns here (and not 3)?